languages/csharp/exercises/concept/constants/.docs/instructions.md

```diff
@@ -0,0 +1,21 @@
+The authentication system that you last saw in (TODO cross-ref-tba) is in need of some attention. You have been tasked with cleaning up the code.
+
+## 1. Set appropriate fields and properties to const
```

Comment:
This is a bit of a technical description, but I must admit that I could not come up with a nice name either. Maybe something like "Ensure that constant values cannot be tampered with"

Reply:
I think that making variables `readonly` is consistent with your wording which itself mentions `const`ants.  I think the test here is to find all the appropriate candidates.  The syntax is trivial.